docs/contributors/guide/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,123 @@
+
+Cherry pick pull requests are reviewed slightly differently than normal pull requests on the `master` branch in that
+they:
+
+- Are by default expected to be `type/bug-fix`.
+
+- The original change to the `master` branch is expected to be merged for some time and no related CI failures or test
+  flakiness must be discovered.
+
+- Milestones must be set on the PR reflecting the milestone for the target release branch (for example, milestone v5.3.2
+  for a cherry pick onto branch
+  `release-5.3`). This is normally done for you by manually.
+
+- A separate cherry pick pull request should be open for every applicable target branch. This ensures that the fix will
+  be present on every active branch for a given set of patch releases. If a fix is only applicable to a subset of active
+  branches, it is helpful to note why that is the case on the parent pull request or on the cherry pick pull requests to
+  the applicable branches.
+
+- Have one additional level of review in that they must be approved specifically for cherry pick by branch approvers.
+
+  Approval is signified by an approver manually applying the
+  `cherry-pick-approved` label. This action removes the
+  `do-not-merge/cherry-pick-not-approved` label and triggers a merge into the target branch.
+
+  If you are concerned about the status of your cherry pick, err on the side of overcommunicating and reach out to the
+  [nongfushanquan](https://github.com/nongfushanquan). We should contact [nongfushanquan](https://github.com/nongfushanquan) as early as possible for approval.
+
+## Searching for Cherry Picks
+
+Examples (based on cherry picks targeting the `release-5.3` branch):
+
+- [`cherry-pick-approved`](https://github.com/pingcap/tiflow/pulls?q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved+base%3Arelease-5.3)
+- [`do-not-merge/cherry-pick-not-approved`](https://github.com/pingcap/tiflow/pulls?q=is%3Aopen+is%3Apr+label%3Ado-not-merge%2Fcherry-pick-not-approved+base%3Arelease-5.3)
+
+## Troubleshooting Cherry Picks
+
+Contributors may encounter some of the following difficulties when initiating a cherry pick.
+
+- A cherry pick PR does not apply cleanly against an old release branch. In that case, you will need to manually fix
+  conflicts.
+
+- The cherry pick PR includes code that does not pass CI tests. Please re-trigger the test and submit an unstable test
+  report [in this issue](https://github.com/pingcap/tiflow/issues/2246).
+
+## Batch Merging Cherry Picks
+
+If you have created a large number of cherry picks for the same module over time, we recommend that you merge these PRs
+in batches.
+
+- Manually merge your cherry pick PRs into a new PR.
+
+  Please use the title of the original cherry pick PR as the commit message for the new PR. For example, if you merge 5
+  PRs, use the titles of these 5 PRs as the 5 commit messages for this new PR.
+
+  Please use `<subsystem>(ticdc|dm|both): rollup cherryp picks` as the title of the PR.
```

Comment:
```suggestion
  Please use `<subsystem>(ticdc|dm|both): rollup cherry picks` as the title of the PR.
```
We should recommend authors to use a meaningful PR title, eg. rollup of Kafka bug fixes.

Reply:
It should be represented right in the subsystem?

Reply:
> It should be represented right in the subsystem?

Yes, it should be a brief summary.

Reply:
I think it might be hard to write clearly in the title what so many commits are doing. Also there will be a brief summary in our PR body. So let's keep it simple here.